ti/savings/savings.go

```diff
@@ -11,21 +11,36 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
-func ParseAndUploadSavings(ctx context.Context, workspace string, log *logrus.Logger, stepID string, tiConfig *tiCfg.Cfg) {
+func ParseAndUploadSavings(ctx context.Context, workspace string, log *logrus.Logger, stepID string,
+	cmdTimeTaken int64, tiConfig *tiCfg.Cfg) {
 	// Cache Savings
 	start := time.Now()
-	state, timeTaken, err := cache.ParseCacheSavings(workspace, log)
+	cacheState, timeTaken, err := cache.ParseCacheSavings(workspace, log)
 	if err == nil {
-		log.Infof("Successfully parsed savings with state %s and time %sms in %0.2f seconds",
-			state, strconv.Itoa(timeTaken), time.Since(start).Seconds())
+		log.Infof("Successfully parsed Build Cache savings with state %s and time %sms in %0.2f seconds",
```

Comment:
s/parsed/computed. Let's not use `savings` here when we are printing the total time taken (not savings). Consider: `Computed build cache execution details. State: , Time taken:`

Reply:
Done